sfa_dash/form_utils/tests/test_converter.py

```diff
@@ -629,3 +629,49 @@ def test_report_converter_apply_crps_crpss(pair, expected_metrics):
     updated = converters.ReportConverter.apply_crps(params)
     assert updated['metrics'] == expected_metrics
 
+
+def test_report_converter_payload_to_formdata_timezone_default(report):
+    form_data = converters.ReportConverter.payload_to_formdata(report)
+    assert form_data['report_parameters']['timezone'] == ""
+
+
+def test_report_converter_payload_to_formdata_timezone_null(report):
+    the_report = deepcopy(report)
+    the_report['report_parameters']['timezone'] = None
+    form_data = converters.ReportConverter.payload_to_formdata(the_report)
+    assert form_data['report_parameters']['timezone'] == ""
+
+
+@pytest.mark.parametrize("tz,expected", [
+    ("", None),
+    ("America/Pheonix", "America/Pheonix")
```

Comment:
is this intentionally misspelled? (Phoenix)

Reply:
hooked on feeownix